app/src/main/java/pro/itshark/moneysplitter/presentation/regLogin/login/LoginViewModel.kt

```diff
@@ -11,10 +12,13 @@ import javax.inject.Inject
 class LoginViewModel
 @Inject constructor(private val userUseCases: UserUseCases): ViewModel() {
     val stateLiveData = MutableLiveData<LoginScreenState>()
-
     val credits = LoginModel()
+    val requestFailed: ObservableField<Boolean> = ObservableField(false)
```

Comment:
`isRequestSuccess` - cause its boolean and in most cases we must to use some positive way for naming

Reply:
Renamed